src/App.js

```diff
@@ -1,27 +1,33 @@
 import React from 'react'
-import Burger from './components/Burger/Burger'
-import Kebab from './components/Kebab/Kebab'
-import Minced from './components/Minced/Minced'
-import Nuggets from './components/Nuggets/Nuggets'
-import Sausages from './components/Sausages/Sausages'
+import ProductOverview from './components/ProductOverview/ProductOverview'
+import { Switch, Route } from 'react-router-dom'
+import Burger from './pages/Burger/Burger'
+import Kebab from './pages/Kebab/Kebab'
+import Minced from './pages/Minced/Minced'
+import Nuggets from './pages/Nuggets/Nuggets'
+import Sausages from './pages/Sausages/Sausages'
+import { ReactComponent as Logo } from './images/icons/logo.svg'
 import styled from 'styled-components'
+import productCategories from '../src/mocks/category.json'
 
-export default function App() {
+export default function App(){
     return(
-        <>
-        <HeadlineStyled>All vegan products</HeadlineStyled>
-        <Burger />
-        <Kebab />
-        <Minced />
-        <Nuggets />
-        <Sausages />
-        </>
+            <Switch>
+                <Route exact path="/"> 
+                <LogoStyled />
+                {productCategories.map((category, index) => 
+                <ProductOverview key={index} productIcon={category.icon} productName={category.name}/>)}
+                </Route>
+                <Route path="/burger" component={Burger}/>
+                <Route path="/kebab" component={Kebab}/>
+                <Route path="/minced" component={Minced}/>
+                <Route path="/nuggets" component={Nuggets}/>
+                <Route path="/sausages" component={Sausages}/>
```

Comment:
You currently don't have a route for cold-cuts, when I click on the icon, I get referred to a blank page

Reply:
I currently have no data for it. I guess I should put a "Coming soon" in there as long as there is nothing to show.